### DISCUSSION TITLE
Data quality

### DISCUSSION THREAD

**Discussion by iBobik** [teambtcmap/btcmap.org#72] [Ideas]
Hi,
I evangelize Bitcoin payments in Czechia and I was faced one problem - bitcoiners does not know where they can pay. Coinmap is not useable and has too much old places or staff does not know about Bitcoin, so after few tries they stop asking and keep HODLing.

So we brainstormed and interviewed local community how to solve it - so holders will pay in restaurants and businesses. Result was they need to know where they can pay for sure - any time there will be staff who knows how to accept LN payment and it will highly probably work.

So I started non-profit consultations for businesses - www.berubitcoin.cz

and map with places from Coinmap (grey) and verified places (orange) - mapa.berubitcoin.cz

Verified means some independent community member mystery shopped there in the last half year. If there will be issues (most common is only business owner has wallet in his phone), we offer them help.

Does it make sense for you? Would you like to implement this verification process?

I can offer help and also my map's frontend. Ideally I like to merge my local map project with btcmap.org.
https://github.com/iBobik/berubitcoin

**Comment by bubelov** [comment]
Looks good, we can do manual merge. Consolidating all the data sources via OSM makes sense. It's a well-maintained and rapidly growing platform managed by a non-profit org, so all Bitcoin map frontends can share the same data set and work on it together. We have an open API so you can also query our verified places in case you want to keep your own frontend and display them there

**Comment by dadofsambonzuki** [comment]
Thanks for getting in touch.

As Igor says, we are using OSM as the database and encouraging local user
to verify the data in there to make sure it is current.

Check out the tagging and verification process here:

https://github.com/teambtcmap/btcmap-data/wiki/Tagging-Instructions

It would be great if you could switch to using OSM and then either using
their API, our API or embedding our map directly (
https://github.com/teambtcmap/btcmap.org#embedding).

Let us know if you need any help.

**Comment by iBobik** [comment]
Hi, mainreason for me to create my map was to filter out inactive and low quality places. Like when they accept only on owner's phone or most of the staff does not know they ever accept Bitcoin or missing Lightening.In the Btcmap I does not see easy way how to report places what should not be in the map or some process helping reported business.I would like to stop maintaining redundant app or at least it's data, but replacement should be better. 🙂Honza Pobořil💬Signal13. 1. 2023 v 20:24, Nathan Day ***@***.***>:﻿
Thanks for getting in touch.

As Igor says, we are using OSM as the database and encouraging local user
to verify the data in there to make sure it is current.

Check out the tagging and verification process here:

https://github.com/teambtcmap/btcmap-data/wiki/Tagging-Instructions

It would be great if you could switch to using OSM and then either using
their API, our API or embedding our map directly (
https://github.com/teambtcmap/btcmap.org#embedding).

Let us know if you need any help.

**Comment by iBobik** [comment]
How exactly works your reporting process on the web? Could it make sence to
include it in my frontend or better have my own, so I can receive reports
in Czech language and offer local consultancy?

13. 1. 2023 7:04:49, Igor Bubelov ***@***.***> napsal:

**Comment by bubelov** [comment]
@iBobik you can click on any place and hit the "Verify" button. It's also used for reporting outdated places, but I think we need to make it clearer, probably by separating verification and reports. We're automatically creating a data repo issue for every report.

https://github.com/teambtcmap/btcmap-data/issues?q=is%3Aissue+is%3Aopen+label%3ACzechia

I assume Czechia and Czech Republic are the same country (correct me if I'm wrong). Every issue has a country tag and all of those issues are being handled by the volunteer OSM mappers. It would be nice if you can join efforts and take ownership of your region. Locals are always preferred because they know local language and have more local knowledge. Currently, some of the most active contributors have to do non-local edits, but it's only because we don't have enough local leads and we don't want to have stale issues, so it's a temporary thing.

@secondl1ght perhaps we can auto-assign geofenced issues to the community/country leads? That would prevent other non-local mappers from handling those issues. Locals know best =)

**Reply by secondl1ght** [reply]
I think this is a good idea in the future if we get more local leaders that are contributing to maintaining the map, currently there aren't many right?

**Reply by bubelov** [reply]
> I think this is a good idea in the future if we get more local leaders that are contributing to maintaining the map, currently there aren't many right?

The problem is, other taggers don't know whether a specific area has a lead, so they can handle it faster than locals. It's not ideal, and perhaps we can tell the taggers not to touch the issues assigned to other users. Issues for unmaintained areas would stay unassigned and signal other mappers that they can be picked up.

**Reply by secondl1ght** [reply]
Yes that's a good point, but currently there are only about 5 or so active taggers I think. And these taggers already prioritize their local areas if possible. If we auto assign issues then they might get completed slower or not at all if we don't have an up-to-date list of who the active maintainers are. We would have to maintain a list of community leaders and make sure their tasks are getting completed so they aren't being held for no reason when somebody else might have taken them on. Also not all taggers have GH accounts either. It's a cool idea but might get stale due to quick tagger turnover and then be counter productive.

**Comment by dadofsambonzuki** [comment]
The main tag we use to determine good data is the survey:date tag. Anything
surveyed within the last 12-months we consider up-to-date.

Yeah, we've talked about emailing the community lead when a new Issue is
raised in their area.

We could expand this to enable specific OSM users to join specific
communities and subscribe to a range of notifications.

Sounds like we should create you a Czechia Community!

btcmap.org/community/add

Don't worry about the map boundaries, we'll use a GeoJSON of Czechia.

What do you think?

**Comment by dadofsambonzuki** [comment]
We can put that community info on the Issue and label it differently. We
then guide the supertaggers to lower the priority of these in order to help
distribute responsibility to locals.

Nathan Day

e: ***@***.***
m: +44 7982 234 581

**Reply by secondl1ght** [reply]
I think having the country labels is already enough to get locals to prioritize their own issues. Adding community info would be a bit redundant and also get messy quick since communities can overlap eachother. We don't have a problem of too many taggers right now, we are barely getting through the issues as-is. It might help the issues get more visibility though if we add them to the Website, they could have their own page and be grouped by country?

**Comment by dadofsambonzuki** [comment]
Yep, definitely needs to wait until we have all the communities
bootstrapped.

**Comment by dadofsambonzuki** [comment]
Yeah, I think the new and verify issues should be listed on either the
Community and/or Country page they fall within.

This would be a good first step and we can deal with notifications later.

**Reply by secondl1ght** [reply]
We would have to test and see how resource intensive checking for point contains on every community for every noob submission would be. If it's not too bad we could do it, countries are just a single API geocoding call which we already have setup. I think even having an Open Issues page on the website would be good. Maybe I should implement that even before we get the countries setup because that is waiting on data.
